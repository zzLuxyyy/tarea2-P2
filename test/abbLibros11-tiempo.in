obtenerExisteLibroABBLibrosTiempo 200000 0.01
Fin
