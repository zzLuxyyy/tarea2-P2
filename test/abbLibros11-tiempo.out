1>Se obtuvieron los libros 1 1 1 1 con ISBNS respectivos 0 199999 66666 133333
Calculado correctamente en menos de 0.010s.
2>Fin.
